gradle/libs.versions.toml

```diff
@@ -50,6 +50,7 @@ pluginPackages-spotless = "com.diffplug.spotless:spotless-plugin-gradle:6.10.0"
 pluginPackages-taskTree = "com.dorongold.plugins:task-tree:2.1.0"
 pluginPackages-versionCatalogUpdate = "nl.littlerobots.version-catalog-update:nl.littlerobots.version-catalog-update.gradle.plugin:0.5.3"
 pluginPackages-versionsFilter = "se.ascp.gradle:gradle-versions-filter:0.1.16"
+pluginPackages-javaLibrary = "???"
```

Comment:
`javaLibrary` is a bundled/built-in plugin, shipped together with Gradle itself; I'd guess there isn't a need to include it in version catalog... or is there? 🤔

Reply:
JavaLibrary is built-in, but I can't find the DSL